components/commons/avatarOrganization.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <div class="circle">
+    <img src="example.png">
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'AvatarOrganization'
```

Comment:
A component must be abstract and abroad a lot of cases. This component can be named only `Avatar` so it can be used in other places for other reasons.

It's always good to not specialize components.